### ISSUE TITLE
Registry Object Not Present  [CRASH]

### ISSUE THREAD

**Issue by AtlasX20** [andrew0030/Pandoras-Creatures#14]
Downloaded newest version of the mod today, game crashed with this error before startup. 

**Crash log**
https://pastebin.com/f68C2qh6
**Screenshots**
![Screenshot 2020-09-24 183241](https://user-images.githubusercontent.com/71855640/94210446-5d2bfc00-fe94-11ea-96ef-3243c8bbd8b4.gif)

**Versions**
1.15.2-1.0.1

**Comment by andrew0030** [comment]
I think I know whats causing this error, the issue is most likely happening because of this here:
https://github.com/andrew0030/Pandoras-Creatures/issues/13
Im not 100% sure so I wont add the duplicate label for now.
I will revisit 1.14 and 1.15 to fix it, as soon as Im done updating to 1.16

**Comment by andrew0030** [comment]
I looked into it and I dont think this has to do with issue #13 after all. I recall somebody telling me that their game crashed when running Pandoras Creatures and Random Enchants. I did try to replicate this issue by running the latest version of Random Enchants and Pandoras Creatures, but it seems to work nicely. Try updating both mods if updates are available and if not, try removing random enchants and see if it still crashes.

**Comment by CGV224** [comment]
I'm having this exact same issue.

**Comment by andrew0030** [comment]
I will need a little bit more information then that, what mod version what other mods and so on

**Comment by CGV224** [comment]
Oh I took all of my mods out of my mod folder then put them back in and it worked.

**Comment by andrew0030** [comment]
yeah but still, what mods do you use? Is random enchants on the list?

**Comment by CGV224** [comment]
I'm using 50+ mods and random enchants is not on the list.

**Comment by andrew0030** [comment]
Minecraft version 1.15?

**Comment by CGV224** [comment]
Yeah

**Comment by SkyU12** [comment]
Hey, I'm having same problem as CGV here is the crash log 


[crash-2020-11-09_14.40.44-client.txt](https://github.com/andrew0030/Pandoras-Creatures/files/5510877/crash-2020-11-09_14.40.44-client.txt)
